projects/knative/knative-graduation-dd.md

```diff
@@ -0,0 +1,444 @@
+# Knative Graduation Due Diligence
+
+- Link to [Graduation application issue](https://github.com/cncf/toc/issues/1509)
+- Based on the [graduation DD template](https://github.com/cncf/toc/blob/main/operations/toc-templates/template-dd-pr-graduation.md)
+- Project Repo(s): github.com/knative, github.com/knative-extensions
+- Project Site: https://www.knative.dev
+- Sub-Projects: Serving, Eventing, Functions, Client
+- Communication: https://cloud-native.slack.com/ #knative and various channels with #knative- prefix
+- Project points of contacts: steering@knative.team
+
+## Graduation Evaluation Summary for Knative
+
+### Criteria Evaluation
+
+Lin Sun & Jeremy Rickard conducted the due diligence of Knative who applied for graduation. The project has completed the criteria that show its maturity at graduation. The following criteria implementations are noteworthy to call out:
+- Knative is a mature project that are endorsed by multiple adopters.
+- The project is not only vendor neutral but also has a very diverse set of maintainers, adopters and integrators. The steering committee is composed of members from 4 different set of organizations.
+- Knative documentation is fairly clear and easy to follow.
+- The project does an excellent job of making sure that its working group public meetings are well documented.
```

Comment:
Are there architectural or technical specifics that identify this project as being mature? What exactly makes this project a mature project and what are the trends you see as to why adopters are adopting this project?

Reply:
Yep expanded the doc section on what I observed, adding docs related to upgrade, troubleshooting but also administrator topics such as [configuring HA for serving](https://knative.dev/docs/serving/config-ha/) or [configuring application security with security guard](https://knative.dev/docs/serving/app-security/security-guard-about/)

Reply:
also added project high velocity data below.  Let me know if this addresses your comments, thank you!!